plugins/dup.yaml

```diff
@@ -3,48 +3,41 @@ kind: Plugin
 metadata:
   name: dup
 spec:
-  version: v0.2.0
+  version: v0.3.0
   homepage: https://github.com/vash/dup
-  shortDescription: Duplicate existing Kubernetes resources
+  shortDescription: Easily and powerfully duplicate (and edit) existing Kubernetes resources and pods that are spawned by them.
```

Comment:
This will be truncated heavily. Limit to 60 chars. Let me know if you want me to close so you can manually edit, or just re-tag.

Reply:
In second thought, I'll keep the wording as it was, thanks!

Reply:
What should I do? Close or merge?